pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/ElasticTestUtil.java

```diff
@@ -0,0 +1,208 @@
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.Gson;
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.dsm.db.OncHistoryDetail;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.db.dto.ddp.participant.ParticipantDto;
+import org.broadinstitute.dsm.model.elastic.Profile;
+import org.broadinstitute.dsm.model.elastic.export.painless.UpsertPainless;
+import org.broadinstitute.dsm.util.export.ElasticSearchParticipantExporterFactory;
+import org.broadinstitute.dsm.util.export.ParticipantExportPayload;
+import org.broadinstitute.lddp.handlers.util.Institution;
+import org.broadinstitute.lddp.handlers.util.InstitutionRequest;
+import org.elasticsearch.action.admin.indices.delete.DeleteIndexRequest;
+import org.elasticsearch.action.get.GetRequest;
+import org.elasticsearch.action.get.GetResponse;
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.client.indices.CreateIndexRequest;
+import org.elasticsearch.client.indices.CreateIndexResponse;
+import org.elasticsearch.client.indices.PutMappingRequest;
+import org.elasticsearch.index.query.BoolQueryBuilder;
+import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.xcontent.XContentType;
+import org.junit.Assert;
+
+@Slf4j
+public class ElasticTestUtil {
+
+    public static String createIndexWithMappings(String realm, String mappingsFile) {
+        String indexName = createIndex(realm);
+        try {
+            String mappingsJson = TestUtil.readFile(mappingsFile);
+            updateMapping(indexName, mappingsJson);
+        } catch (Exception e) {
+            e.printStackTrace();
+            Assert.fail("Unexpected exception reading mappings: " + e);
+        }
+        return indexName;
+    }
+
+    public static String createIndex(String realm) {
+        String indexName = String.format("participants_structured.%s", realm);
+        try {
+            RestHighLevelClient client = ElasticSearchUtil.getClientInstance();
+            CreateIndexRequest req = new CreateIndexRequest(indexName);
+            String settingsJson = TestUtil.readFile("elastic/indexSettings.json");
+            req.settings(settingsJson, XContentType.JSON);
+            CreateIndexResponse createIndexResponse = client.indices().create(req, RequestOptions.DEFAULT);
+            log.info("CreateIndexResponse: {}", createIndexResponse);
+        } catch (Exception e) {
+            e.printStackTrace();
+            Assert.fail("Unexpected exception creating index: " + e);
+        }
+        return indexName;
+    }
+
+    public static void deleteIndex(String esIndex) {
+        try {
+            RestHighLevelClient client = ElasticSearchUtil.getClientInstance();
```

Comment:
Do we want some additional safeguards here so that destructive operations like wiping out an index can only be done on disposable ES containers?